diff -Nur libvisual-0.2.0.org/libvisual/lv_cpu.c libvisual-0.2.0/libvisual/lv_cpu.c
--- libvisual-0.2.0.org/libvisual/lv_cpu.c	2005-02-09 19:20:12.000000000 +0000
+++ libvisual-0.2.0/libvisual/lv_cpu.c	2005-02-14 22:41:24.466371736 +0000
@@ -154,21 +154,7 @@
 #else /* !VISUAL_OS_DARWIN */
 	/* no Darwin, do it the brute-force way */
 	/* this is borrowed from the libmpeg2 library */
-	signal (SIGILL, sigill_handler);
-	if (sigsetjmp (__lv_powerpc_jmpbuf, 1)) {
-		signal (SIGILL, SIG_DFL);
-	} else {
-		__lv_powerpc_canjump = 1;
-
-		asm volatile
-			("mtspr 256, %0\n\t"
-			 "vand %%v0, %%v0, %%v0"
-			 :
-			 : "r" (-1));
-
-		signal (SIGILL, SIG_DFL);
-		__lv_cpu_caps.hasAltiVec = 1;
-	}
+	__lv_cpu_caps.hasAltiVec = 1;
 #endif
 }
 #endif
