--- libvisual-0.2.0.org/libvisual/lv_cpu.c      2005-02-09 19:20:12.000000000 +0000
+++ libvisual-0.2.0/libvisual/lv_cpu.c  2005-02-14 22:22:15.940974136 +0000
@@ -154,21 +154,7 @@
 #else /* !VISUAL_OS_DARWIN */
        /* no Darwin, do it the brute-force way */
        /* this is borrowed from the libmpeg2 library */
-       signal (SIGILL, sigill_handler);
-       if (sigsetjmp (_lv_powerpc_jmpbuf, 1)) {
-               signal (SIGILL, SIG_DFL);
-       } else {
-               _lv_powerpc_canjump = 1;
-
-               asm volatile
-                       ("mtspr 256, %0\n\t"
-                        "vand %%v0, %%v0, %%v0"
-                        :
-                        : "r" (-1));
-
-               signal (SIGILL, SIG_DFL);
-               _lv_cpu_caps.hasAltiVec = 1;
-       }
+       _lv_cpu_caps.hasAltiVec = 0;
 #endif
 }
 #endif
@@ -423,7 +409,7 @@
 #endif
 #endif /* VISUAL_ARCH_X86 */

-#if VISUAL_ARCH_POWERPC
+#if defined(VISUAL_ARCH_POWERPC)
        check_os_altivec_support ();
 #endif /* VISUAL_ARCH_POWERPC */
 
